artemis/scheduler.py

```diff
@@ -0,0 +1,60 @@
+from datetime import datetime
+import pytz
+
+from apscheduler.schedulers.background import BackgroundScheduler
```

Comment:
As Artemis can get restarted, it's crucial to store the tasks in a sensible database. Would you mind adding a backend to APScheduler to ensure the jobs are persisted?